config/php-cs-fixer.php

```diff
@@ -2,154 +2,36 @@
-            'no_unused_imports' => true,
-            'ordered_imports' => true,
+            // overwrite the PER2 defaults to restore compatibility with PHP 7.x
+            'trailing_comma_in_multiline' => ['elements' => ['arrays', 'match']],
 
-            // language construct
-            'combine_consecutive_issets' => true,
-            'combine_consecutive_unsets' => true,
-            'declare_equal_normalize' => true,
-            'dir_constant' => true,
-            'is_null' => true,
-
-            // list notation
-            // (no rules used from this section)
-
-            // namespace notation
-            'no_leading_namespace_whitespace' => true,
-
-            // naming
-            // (no rules used from this section)
-
-            // operator
-            'concat_space' => ['spacing' => 'one'],
-            'new_with_braces' => true,
-            'standardize_not_equals' => true,
-            'ternary_operator_spaces' => true,
-            'ternary_to_null_coalescing' => true,
-            'unary_operator_spaces' => true,
-
-            // PHP tag
-            'blank_line_after_opening_tag' => true,
-            'echo_tag_syntax' => true,
-            'linebreak_after_opening_tag' => true,
-
-            // PHPUnit
             'php_unit_construct' => true,
+            'php_unit_dedicate_assert' => ['target' => 'newest'],
+            'php_unit_expectation' => ['target' => 'newest'],
             'php_unit_fqcn_annotation' => true,
+            'php_unit_method_casing' => true,
+            'php_unit_mock' => ['target' => 'newest'],
+            'php_unit_mock_short_will_return' => true,
+            'php_unit_namespaced' => ['target' => 'newest'],
             'php_unit_set_up_tear_down_visibility' => true,
+            'php_unit_test_annotation' => ['style' => 'annotation'],
             'php_unit_test_case_static_method_calls' => ['call_type' => 'self'],
-
-            // PHPDoc
-            'no_blank_lines_after_phpdoc' => true,
-            'no_empty_phpdoc' => true,
-            'phpdoc_add_missing_param_annotation' => true,
-            'phpdoc_indent' => true,
-            'phpdoc_no_package' => true,
-            'phpdoc_scalar' => true,
-            'phpdoc_separation' => true,
-            'phpdoc_trim' => true,
-            'phpdoc_types' => true,
-            'phpdoc_types_order' => ['null_adjustment' => 'always_last', 'sort_algorithm' => 'none'],
-
-            // return notation
-            'no_useless_return' => true,
-
-            // semicolon
-            'multiline_whitespace_before_semicolons' => true,
-            'no_empty_statement' => true,
-            'no_singleline_whitespace_before_semicolons' => true,
-            'semicolon_after_instruction' => true,
-            'space_after_semicolon' => true,
-
-            // strict
-            'declare_strict_types' => true,
-
-            // string notation
-            'escape_implicit_backslashes' => ['single_quoted' => true],
```

Comment:
This should be replaced with `string_implicit_backslashes => ['single_quoted' => 'escape']`, which isn't in any of the rule sets used (`escape_implicit_backslashes` is deprecated).